src/_components/Assets/AddAssetContainer.jsx

```diff
@@ -222,6 +263,18 @@ class AddAssetContainer extends React.Component {
           </div>
 
           <Grid centered columns={2}>
+            <Grid.Row>
```

Comment:
Why not the following?
```jsx
{showStatus && (
  <Grid.Row>
    <Grid.Column>
      <StatusMessageComponent .../>
    </Grid.Column>
  </Grid.Row>
)}
```
What you currently have renders an empty div on the page as shown in the screenshot
![image 2018-11-15 at 11 08 50 am](https://user-images.githubusercontent.com/12892171/48538967-e486ac80-e8c6-11e8-9712-cf4fb7583779.png)

Reply:
I can't seem to replicate this error. But I've made the necessary changes as you've requested.

Reply:
Ohh...you could replicate it by hovering the element in the browser dev tools